src/apps/testapps/testH3Memory.c

```diff
@@ -169,8 +168,7 @@ SUITE(h3Memory) {
                 count++;
             }
         }
-        t_assert(count == expectedCompactCount,
-                 "got expected compactCells count");
+        t_assert(count == expectedCompactCount, "got expected compact count");
```

Comment:
In theory we shouldn't have format only changes with a standardized formatter version - not sure why this is changing. I don't think it needs to be taken out of this PR but something to check on later.

Reply:
👍